src/fiesta/inference/lightcurve_model.py

```
@@ -63,7 +63,7 @@ def get_default_directory(name):
              surrogate_dir = surrogate_dir / "KN" / name / "model"
     
     else:
-        raise ValueError(f"If no model directory is provided, the name for the default models must either start with 'afgpy', 'pbag', 'Bu'.")
+        raise ValueError(f"If no model directory is provided, the name for the default models must either start with 'afgpy', 'pbag', 'jetsimpy', or 'Bu'.")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)